common/src/main/java/io/netty/util/internal/logging/InternalLoggerFactory.java

```diff
@@ -34,6 +36,8 @@
 public abstract class InternalLoggerFactory {
 
     private static volatile InternalLoggerFactory defaultFactory;
+    static final String LINE_NUMBER = "io.netty.loggerLineNumber";
```

Comment:
@chaosky also should this be something that has `slf4j` in the name as its specific to slf4j ?

Reply:
Sorry, I have signed CLA.

Reply:
@chaosky did you see the comment ?